src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -24,7 +24,7 @@ Gets SQL Database capabilities for the current subscription.
 Gets one or more databases.
 
 ### [Get-AzureRmSqlDatabaseActivity](Get-AzureRmSqlDatabaseActivity.md)
-Gets the status of moving elastic databases.
+Gets the status of database operations.
```

Comment:
Does this show both in-progress and recently completed/failed/cancelled?

Reply:
yes